### ISSUE TITLE
ERROR 405 Method Not Allowed

### ISSUE THREAD

**Issue by GranatenArnold** [hydranox/foo-pub#219]
Hallo, eben bekam ich beim Einreichen der Loesung zu Pivot Partitioning folgenden Fehler angezeigt:
![fooerror](https://cloud.githubusercontent.com/assets/7962341/8143992/9a34af66-11c8-11e5-9f5b-c585ab17f517.jpg)

**Comment by hydranox** [comment]
Zufällig dabei die URL editiert?
Du bist in den Schutzmechanismus gerannt, der dafür sorgt, dass man eine Aufgabe nicht zwei mal Abgeben kann.

**Comment by GranatenArnold** [comment]
nein, editiert habe ich nichts. Vielleicht lag es dann an zu langer Inaktivitaet? Also dass die Aufgabe einfach zu lange offen war.
Ich konnte naemlich danach zurueck zur Uebersicht und dieselbe Aufgabe noch einmal von vorn bearbeiten und abgeben.
Wenn das der Fall ist, dann kann dieses Issue natuerlich wieder geschlossen werden ;)
